Example/PaymentSheet Example/PaymentSheet Example/PlaygroundController.swift

```diff
@@ -121,6 +120,19 @@ class PlaygroundController: ObservableObject {
                 state: "California"
             )
         }
+        switch settings.defaultBillingAddress {
+        case .on:
+            configuration.defaultBillingDetails.email = "foo@bar.com"
+            configuration.defaultBillingDetails.phone = "+13105551234"
+        case .randomEmail:
+            configuration.defaultBillingDetails.email = "test-\(UUID().uuidString)@stripe.com"
+            configuration.defaultBillingDetails.phone = "+13105551234"
+        case .randomEmailNoPhone:
+            configuration.defaultBillingDetails.email = "test-\(UUID().uuidString)@stripe.com"
```

Comment:
In efforts of trying to reduce the number of options, is it possible to remove `.on`, and just have:
* randomEmailPhone
* randomEmailNoPhone
* off

?

Reply:
`.on` is nice because it uses a set email address (which is already registered for Link). I'd also like to keep it named `.on` to maintain compatibility with older QR codes.